exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_fix_version.ipynb

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
<p>The specific version of the library used by the <strong>default</strong> SLC depends ...</p><p> </p><p>"AI-Lab adapts its own version of the library if required." I don't think it's a good explanation of what is happening here. AI-Lab itself is not doing anything. The user should run the script like the one in this notebook to make sure the versions are aligned.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I updated the paragraph in the next push.
Please review again.

Reply:
@ahsimb thx, that was also what I meant needs to be formulated differently and @ckunki for changing